src/services/news.data.service.ts

```diff
@@ -30,7 +30,7 @@ export function fetchNews(source = '', offSet = 0, limit = 100): Promise<any> {
   }
   query.push(`offset=${offSet}`);
 
-  query.push('resolution=suburban_road');
+  query.push('resolution=suburban_road&resolution=street');
```

Comment:
we need to verify that this change does not apply  on prod,  something like : 
```suggestion
 if(!isProd()){ 
  query.push(resolution=street');
}

```